docs/tutorials/introductory_tutorials/temporal_graph_classification_pluto.jl

```diff
@@ -0,0 +1,1561 @@
+
+#> [frontmatter]
+#> author = "[Aurora Rossi](https://github.com/aurorarossi)"
+#> title = "Temporal Graph classification with Graph Neural Networks"
+#> date = "2024-03-06"
+#> description = "Temporal Graph classification with GraphNeuralNetworks.jl"
+#> cover = "assets/brain_gnn.gif"
+
+using Markdown
+using InteractiveUtils
+
+# ╔═╡ b8df1800-c69d-4e18-8a0a-097381b62a4c
+begin
+	using Flux
+	using GraphNeuralNetworks
+	using Statistics, Random
+	using LinearAlgebra
+	using MLDatasets: TemporalBrains
+end
+
+# ╔═╡ 69d00ec8-da47-11ee-1bba-13a14e8a6db2
+md"In this tutorial, we will learn how to extend the graph classification task to the case of temporal graphs, i.e., graphs whose topology and features are time-varying.
+
+We will design and train a simple temporal graph neural network architecture to classify subjects' gender (female or male) using the temporal graphs extracted from their brain fMRI scan signals.
+"
+
+# ╔═╡ ef8406e4-117a-4cc6-9fa5-5028695b1a4f
+md"
+## Import
+
+We start by importing the necessary libraries. We use `GraphNeuralNetworks.jl`, `Flux.jl` and `MLDatasets.jl`, among others.
+"
+
+# ╔═╡ 2544d468-1430-4986-88a9-be4df2a7cf27
+md"
+## Dataset: TemporalBrains
+The TemporalBrains dataset contains a collection of functional brain connectivity networks from 1000 subjects obtained from resting-state functional MRI data from the [Human Connectome Project (HCP)](https://www.humanconnectome.org/study/hcp-young-adult/document/extensively-processed-fmri-data-documentation). 
+Functional connectivity is defined as the temporal dependence of neuronal activation patterns of anatomically separated brain regions.
+
+The graph nodes represent brain regions and their number is fixed at 102 for each of the 27 snapshots, while the edges, representing functional connectivity, change over time.
+For each snapshot, the feature of a node represents the average activation of the node during that snapshot.
+Each temporal graph has a label representing gender ('M' for male and 'F' for female) and age group (22-25, 26-30, 31-35, and 36+).
+The network's edge weights are binarized, and the threshold is set to 0.6 by default.
+"
+
+# ╔═╡ f2dbc66d-b8b7-46ae-ad5b-cbba1af86467
+brain_dataset = TemporalBrains()
+
+# ╔═╡ d9e4722d-6f02-4d41-955c-8bb3e411e404
+md"After loading the dataset from the MLDatasets.jl package, we see that there are 1000 graphs and we need to convert them to the `TemporalSnapshotsGNNGraph` format.
+So we create a function called `data_loader` that implements the latter and splits the dataset into the training set that will be used to train the model and the test set that will be used to test the performance of the model.
+"
+
+# ╔═╡ bb36237a-5545-47d0-a873-7ddff3efe8ba
+function data_loader(graphs)
+    dataset = Vector{TemporalSnapshotsGNNGraph}(undef, length(graphs))
+    for i in 1:length(graphs)
+        graph = graphs[i]
+        dataset[i] = TemporalSnapshotsGNNGraph(GraphNeuralNetworks.mlgraph2gnngraph.(graph.snapshots))
```

Comment:
Not for this tutorial, but `mlgraph2gnngraph` could be extended to handle datasets made of temporal graphs, so that we can just do `dataset = mlgraph2gnngraph(brain_dataset)`

Reply:
yes it is in my plans, I will work on it soon!